test/lib/mayaUsd/render/mayaToHydra/CMakeLists.txt

```diff
@@ -16,6 +16,8 @@ set(INTERACTIVE_TEST_SCRIPT_FILES
     testMayaLights.py
     testFlowViewportAPI.py
     testStageVariants.py
+# Skip until MtoA plugin can be loaded.
+#   testArnoldLights.py
```

Comment:
I woul keep it and rather use something in the test. Please see the comment below about :
if (mayaUtils.loadPlugin("mtoa"))